velox/common/caching/SsdFileTracker.cpp

```diff
@@ -39,22 +39,29 @@ std::vector<int32_t> SsdFileTracker::findEvictionCandidates(
     int32_t numCandidates,
     int32_t numRegions,
     const std::vector<int32_t>& regionPins) {
-  // Calculates average score of regions wiht no pins. Returns up to
+  // Calculates average score of regions with no pins. Returns up to
   // 'numCandidates' unpinned regions with score <= average, lowest
   // scoring region first.
-  int64_t scoreSum = 0;
   int32_t numUnpinned = 0;
   for (int i = 0; i < numRegions; ++i) {
     if (regionPins[i] > 0) {
       continue;
     }
     ++numUnpinned;
-    scoreSum += regionScores_[i];
   }
   if (numUnpinned == 0) {
     return {};
   }
-  const auto avg = scoreSum / numUnpinned;
+  uint64_t runningAvg = 0;
```

Comment:
If this is to avoid overflow, then please use double to ease the implementation. Please verify the score read from checkpoint is as expected. Thanks!